src/app/search/search-results-item-list/search-results-item-list.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-search-results-item-list',
+  templateUrl: './search-results-item-list.component.html',
+  styles: ['mat-grid-list {width: 90%; left: 5%;}']
+})
```

Comment:
change detection strategy could be changed to `OnPush`

https://blog.thoughtram.io/angular/2016/02/22/angular-2-change-detection-explained.html